src/components/Banner/AnonBanner/AnonBannerA.module.scss

```diff
@@ -0,0 +1,7 @@
+@import '../../../../node_modules/@santiment-network/ui/variables';
+
```

Comment:
Current styles do not match base styles in [master](https://github.com/santiment/app/blob/a5ea90525cc3f631177f5641646c699d80ebde0b/src/components/AnonBanner/AnonBannerA.module.scss#L2)